chain-impl-mockchain/src/update.rs

```diff
@@ -243,42 +199,32 @@ mod tests {
             }
         }
     }
-    impl Arbitrary for SettingsDiff {
-        fn arbitrary<G: Gen>(g: &mut G) -> SettingsDiff {
-            SettingsDiff {
-                block_id: ValueDiff::Replace(Arbitrary::arbitrary(g), Arbitrary::arbitrary(g)),
-            }
-        }
-    }
     impl Arbitrary for BftSelectionDiff {
         fn arbitrary<G: Gen>(g: &mut G) -> BftSelectionDiff {
             BftSelectionDiff {
                 leader: ValueDiff::Replace(Arbitrary::arbitrary(g), Arbitrary::arbitrary(g)),
             }
         }
     }
-    impl Arbitrary for LeaderSelectionDiff {
-        fn arbitrary<G: Gen>(g: &mut G) -> LeaderSelectionDiff {
-            LeaderSelectionDiff {
-                bft: Arbitrary::arbitrary(g),
-            }
-        }
-    }
 
     quickcheck! {
+        /*
```

Comment:
Dead code, or should there be a FIXME comment stating why is it commented out?

Reply:
Comment added.